pandas/tests/groupby/aggregate/test_aggregate.py

```diff
@@ -203,6 +203,18 @@ def test_aggregate_str_func(tsframe, groupbyfunc):
     tm.assert_frame_equal(result, expected)
 
 
+def test_agg_str_with_kwarg_axis_1_raises(df, reduction_func):
+    gb = df.groupby(level=0)
+    if reduction_func in ("idxmax", "idxmin"):
```

Comment:
is this new?

Reply:
With regards to idxmax/idxmin, no. Those are the two reduction_func that support axis=1 (whereas all others do not). However, they fail here because `df` has string and numeric columns.